my-app/src/App.css

```diff
@@ -0,0 +1,28 @@
+*{
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```

Comment:
You can remove any font styles from App.css as index.css contains the global font settings.

Reply:
Done